Create Large Test Files.py

```diff
@@ -0,0 +1,255 @@
+        with open("edits_files/lar_rows.txt", 'w') as out_lar: #Writes a file of LAR data.
+            out_lar.writelines(lar_rows) 
+        with open("edits_files/ts_row.txt", 'w') as out_ts: #Writes a file of TS data. 
+            out_ts.writelines(ts_row)
+        with open('schemas/lar_schema.json') as d: #Loads in the schema for the LAR file as a dataframe.
+            data = json.load(d)
+            data = pd.DataFrame(data)
+
+        #Reads in the LAR data as a dataframe, as object values without converting NA values to 
+        #Not a Number Values. 
+        df = pd.read_csv('edits_files/lar_rows.txt', delimiter = "|", header = None, dtype = 'object',
+                         keep_default_na=False)
+        df.columns = data['field']
+        
+        #Concatenates data to produce the desired number of rows in a new LAR dataframe. 
+        df = pd.concat([df]*int((count/1000)), ignore_index = True)  
+
+        #Creates a list of unique ULI's for Bank 1, and checks to determine that ULI's are unique.
+        uli_1 = {'uli':list(['BANK1LEIFORTEST12345']*int((count)))}
+        df1 = pd.DataFrame(uli_1)
+        df1["uli"] = df1.apply(lambda x: x.uli + char_string_gen(23), axis = 1)
+        df1['uli'] = df1.apply(lambda x: x.uli + check_digit_gen(ULI = x.uli), axis = 1)
+        uli_check(df1)
+        
+        #Replaces ULI's in the new LAR dataframe with the unique ULI's.  
+        df['uli'] = df1['uli']
+        
+        #Creates a new LAR csv file with the new LAR data with unique ULI's. 
+        df.to_csv("lar_" + str(count) + ".txt", sep = "|", index = False, header= None)
+
+        #The following appends the new LAR data into the TS file created earlier. 
+        #The new file is renamed "clean_file_(count)_row_Bank1.txt."
+        #New clean files are placed in the "edits_files/clean_files/bank1/" directory.
+        
+        filepath = "edits_files/clean_files/bank1/"
+        os.makedirs(os.path.dirname(filepath), exist_ok=True)
+        
+        with open("lar_" + str(count) + ".txt", 'r' ) as f:
+            lar_count = f.readlines()
+
+        with open("edits_files/ts_row.txt", 'a') as append_lar:
+            append_lar.writelines(lar_count)
+
+        os.remove("lar_" + str(count) + ".txt")
+        os.rename("edits_files/ts_row.txt", "edits_files/clean_files/bank1/clean_file_" + 
+                         str(count) + "_rows_Bank1.txt")
+        return print(str("{:,}".format(count)) + " Row File Created for Bank " + str(bank))
+
+    else:
+        statement = "Please list bank as '0' or '1'"
+        return print(statement)
+        
+    
+
+
+# In[300]:
+
+
+large_clean_test_files(bank = 1, count = 3000)
+
+
+# ### Creating Clean Test Files With Specified Counts in a List
+# A loop may be used to create clean large test files with the counts in a list. 
+
+# In[303]:
+
+
+counts = [1000, 5000, 10000, 50000, 100000]
```

Comment:
This piece of code would work well as a function inside a class. Let's work out a refactor so that this is a stand alone class and calls other classes and class methods where code exists for them.